dspace-api/src/main/java/org/dspace/discovery/IndexClient.java

```diff
@@ -150,10 +168,55 @@ public static void main(String[] args) throws SQLException, IOException, SearchS
             indexer.optimize();
         } else if (line.hasOption('s')) {
             checkRebuildSpellCheck(line, indexer);
+        } else if (line.hasOption("item_uuid")) {
+            String itemUUID = line.getOptionValue("item_uuid");
+            Item item = ContentServiceFactory.getInstance().getItemService().find(context, UUID.fromString(itemUUID));
+            indexer.indexContent(context, item, line.hasOption("f"));
+        } else if (line.hasOption("u")) {
+            String optionValue = line.getOptionValue("u");
+            String[] identifiers = optionValue.split("\\s*,\\s*");
+            for (String id : identifiers) {
+                if (id.startsWith(ConfigurationManager.getProperty("handle.prefix")) || id.startsWith("123456789/")) {
+                    BrowsableObject dso = (BrowsableObject) HandleServiceFactory.getInstance()
+                            .getHandleService().resolveToObject(context, id);
+                    indexer.indexContent(context, dso, line.hasOption("f"));
```

Comment:
Best to add a check to see that the object was found, if it wasn't found a NullPointer will follow.

Reply:
option not needed, removed (it was already supported by the -i )